### PR TITLE
Release 0.8.0 part 1/3: CHANGELOG, README, Makefile

### PR DESCRIPTION
**Pull request by khoang98**
**Is this a bug fix or adding new feature?**
Bug fix

**What is this PR about? / Why do we need it?**
Update AL2 base image again & create release with updated helm

**What testing is done?** 
None

### CODE REVIEW

File: CHANGELOG-0.x.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# v0.8.0
+
+### Misc.
+* Release 0.7.0 part 3/3 ([#224](https://github.com/kubernetes-sigs/aws-fsx-csi-driver/pull/224), [@khoang98](https://github.com/khoang98))
```
Comment (wongma7) [inline CHANGELOG-0.x.md:4]: I would remove these but at the same time I don't mind keeping the list exhaustive and just keeping this stuff under 'Misc.' either

Comment (khoang98) [reply CHANGELOG-0.x.md:4]: Ah ok makes sense. Will leave for now but if it gets too unreadable, can remove these in future

**Pull request merged**